python/samples/getting_started/04-kernel-arguments-chat.ipynb

```diff
@@ -190,7 +173,10 @@
    "source": [
     "from semantic_kernel.functions import KernelArguments\n",
     "\n",
-    "arguments = KernelArguments(user_input=\"Hi, I'm looking for book suggestions\", history=chat_history)"
+    "chat_history = \"\"\"\n",
```

Comment:
I don't think this is what we're trying to convey with this chat history concept. Why does this need to be changed? On a concrete chat history object one can called `add_user_message()`. Why not just do that? It's much easier than someone trying to figure out how to write this in XML.

Reply:
The reason of the change is that this is an anti-pattern, because if someone wants to build a simple chat experience with the chat history object, they don't need to use the prompt template. I agree we should have a separate example showing how to use the chat history class.

Also, this aligns with what .Net has: https://github.com/microsoft/semantic-kernel/blob/main/dotnet/samples/GettingStarted/Step5_Chat_Prompt.cs

Reply:
Let's introduce a new sample that follows the pattern of the dotnet one, where we can have some explanation of the xml internals.